GOVERNANCE.md

```diff
@@ -22,30 +18,226 @@ New maintainers must be nominated by an existing maintainer and must be elected
 
----
-# Proposal Process 
-The proposal process is currently being worked on. No formal process is available at this time. You may reach out to the maintainers in the Kubernetes Slack Workspace within the [#pinniped](https://kubernetes.slack.com/archives/C01BW364RJA) channel or on the [Pinniped mailing list](project-pinniped@googlegroups.com) with any questions you may have or to send us your proposals.
-
----
-# Lazy Consensus
-To maintain velocity in Pinniped, the concept of [Lazy Consensus](http://en.osswiki.info/concepts/lazy_consensus) is practiced. Ideas and / or proposals should be shared by maintainers via GitHub. Out of respect for other contributors, major changes should also be accompanied by a ping on the Kubernetes Slack in [#Pinniped](https://kubernetes.slack.com/archives/C01BW364RJA) or a note on the [Pinniped mailing list](project-pinniped@googlegroups.com) as appropriate. Author(s) of proposals for major changes will give a time period of no less than five (5) working days for comment and remain cognizant of popular observed world holidays.
+## Supermajority
+A supermajority is defined as two-thirds of members in the group. A supermajority of maintainers is required for certain decisions as outlined in this document. A supermajority vote is equivalent to the number of votes in favor being at least twice the number of votes against. A vote to abstain equals not voting at all. For example, if you have 5 maintainers who all cast non-abstaining votes, then a supermajority vote is at least 4 votes in favor. Voting on decisions can happen on the mailing list, GitHub, Slack, email, or via a voting service, when appropriate. Maintainers can either vote "agree, yes, +1", "disagree, no, -1", or "abstain". A vote passes when supermajority is met.
 
-**What constitutes the need for a proposal?**
-If there is significant risk with a potential feature or track of work (such as complexity, cost to implement, product viability, etc.), then we recommend creating a proposal for feedback and approval. If a potential feature is well understood and doesn't impose risk, then we recommend a **standard GitHub issue** to clarify the details.
+## Lazy Consensus
+To maintain velocity in Pinniped, the concept of [Lazy Consensus](http://en.osswiki.info/concepts/lazy_consensus) is practiced.
 
 Other maintainers may chime in and request additional time for review, but should remain cognizant of blocking progress and abstain from delaying progress unless absolutely needed. The expectation is that blocking progress is accompanied by a guarantee to review and respond to the relevant action in short order.
 
 Lazy consensus does not apply to the process of:
 * Removal of maintainers from Pinniped
 
+## Updating Governance
+All substantive changes in Governance, including substantive changes to the proposal process, require a supermajority agreement by all maintainers.
+
+# Proposal Process
```

Comment:
I'm thinking this whole section lives in a 'Proposals' folder like https://github.com/vmware-tanzu/carvel/tree/develop/proposals

and within the governance doc, it's simply linked out like here https://github.com/vmware-tanzu/carvel/blob/develop/GOVERNANCE.md#proposal-process

Reply:
I haven't done this yet, but I'm planning to do it as the last step before merging this PR.